src/components/forgetPass/resetPassword.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import * as image from '../image/image.js';
+import { useSelector, useDispatch } from 'react-redux';
+import * as actionsReset from '../../actions/axiosResetPass.js';
+import { useLocation } from 'react-router-dom';
+import * as message from '../message.js';
+import AlertForm from '../alert/alert.js';
+
+function ResetPassword(props) {
+  const dispatch = useDispatch();
+  const queryString = require('query-string');
+  const headersUrl = queryString.parse(useLocation().search);
+
+  const [displayAlert, setDisplayAlert] = useState('none');
+  const [password,setPassword] = useState('');
+  const [passConf ,setPassConf] = useState('');
+
+  const clickToSend = (event) => {
+    event.preventDefault();
+    if(password === passConf && password !== ''&& passConf !== ''){
+      dispatch(actionsReset.sendResetPassAPI(password,  headersUrl))
+    }else{
+      setDisplayAlert('');
+    }
+    
+  }
+
+  return (
+    <div className='container' style={{ width: '40%' }}>
+      <AlertForm  displayAlert={displayAlert} setDisplayAlert={setDisplayAlert} message={message.messageResetPass}/>
```

Comment:
component nào cũng phải cho AlertForm vào thì ko ổn :-?
em nên để AlertForm vào component to nhất